<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Graphics</class>
 <widget class="QDialog" name="Dialog_Graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="grid_layout_figures">
       <property name="spacing">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QChartView" name="chartview_1"/>
       </item>
       <item row="0" column="1">
        <widget class="QChartView" name="chartview_2"/>
       </item>
       <item row="0" column="2">
        <widget class="QChartView" name="chartview_3"/>
       </item>
       <item row="1" column="0">
        <widget class="QChartView" name="chartview_4"/>
       </item>
       <item row="1" column="1">
        <widget class="QChartView" name="chartview_5"/>
       </item>
       <item row="1" column="2">
        <widget class="QChartView" name="chartview_6"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_buttons">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_start_graphics">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
